YouVsVirus/Assets/Scripts/Components/NPC.cs

```diff
@@ -49,10 +54,24 @@ void FixedUpdate()
         {
             if (CanMove())
             {
-                RandomMovement();
+                // in the demo level the NPCs move from right to left
+                if (LevelSettings.GetActiveSceneName() == "YouVsVirus_Leveldemo")
```

Comment:
I am really not a fan of these `if levelname == STRING`, since

a) it makes it more difficult to use the feature in other levels

b) if we rename the scene we have to find all occurences in the code and change the string appropriately, and we wont get notified by the compiler. We could just forget one and the game would simply behave differently.

I would prefer if this would be a setting of the class that can be activated from outside.
In this case, maybe the CreateHumans class could have a public bool A and the NPC also gets a bool B.
If then the CreateHumans bool is set in the unity scene editor it activates the NPC bool for the NPCs it creates.

Reply:
Yes, I totally agree. However, there are several instances in the code where this is used (maybe not many, maybe these + 1 or 2 ? ). I would say that we postpone this to after the release and then make it our mission to remove all string compares for the levels.

Reply:
ok